Frontend/pages/products.js

```diff
@@ -2,22 +2,32 @@ import React from 'react'
 import Container from '@mui/material/Container';
 import Grid from '@mui/material/Grid';
 import Paper from '@mui/material/Paper';
+import Head from 'next/head'
 import ProductsForm from "../components/products/productsForm";
 import ProductsTable from "../components/products/productsTable";
 
 const Products = () => {
     return (
         <div>
+        <Head>
+            <title>Products</title>
+        </Head>
         <Container maxWidth="lg" sx={{ mt: 4, mb: 4 }}>
             <Grid container spacing={3}>
                 <Grid item xs={12}>
-                    <Paper sx={{ p: 2, display: 'flex', flexDirection: 'column' }}>
+                    <Paper
+                        sx={{
+                            p: 2,
+                            display: "flex",
+                            flexDirection: "column"
+                        }}
+                    >
                         <ProductsForm />
                     </Paper>
                 </Grid>
                 <Grid item xs={12}>
-                    <Paper sx={{ p: 2, display: 'flex', flexDirection: 'column' }}>
-                        <ProductsTable/> 
+                    <Paper sx={{ p: 2, display: "flex", flexDirection: "column" }}>
```

Comment:
<img width="520" alt="image" src="https://user-images.githubusercontent.com/89083753/174142879-ff077afa-263b-43c7-af3b-a02ea5e3cf09.png">
I did it but I received some weird error messages.... Can you explain me for that?

Reply:
I meant i wanted to commit and then received that lol

Reply:
1. Changes not stages for commit: Those are changes that are not in staging, which means they won't be included in the commit. You could use git add <file-name> to stage them and commit them. I haven't use Github Desktop yet so I don't know how much it differs from cli.
2. It would be great to include node_modules to the .gitignore file :)

Reply:
Nope i just worked with git in terminal

Reply:
Done with commit "uniform format"